src/components/pager/pager.style.js

```diff
@@ -109,9 +109,8 @@ const StyledPagerNoSelect = styled.span`
 `;
 
 const StyledPagerSummary = styled.div`
-  display: flex;
   flex: 1 1 30%;
-  justify-content: flex-end;
+  text-align: right;
```

Comment:
This will slightly alter how it looks when the text wraps at smaller screens so might need UX to approve just to be sure (I'm happy to approve the change though). 

Updated behaviour:
![image](https://user-images.githubusercontent.com/44157880/177792449-d637d34d-9ac3-484b-ade3-993298fe7eb7.png)

Current behaviour :
![image](https://user-images.githubusercontent.com/44157880/177792724-1594c7b8-4497-4be5-bb69-31cdb7d6d7fd.png)



Looking at the html for this it can do with a real tidy up (I suspect Git BLAME will say it was me 🤦 so don't look), we have a div rendered empty when the page size selector isn't rendered and multiple containers for the elements are flexed when we can probably just rely on the Pager container handling the flex and layout and would likely negate the needing a `StyledPagerSummary` (a standard div would work). However, all this can be tidied up in a separate ticket as we have one on the backlog to review and fix the html for Pager.

Reply:
good point, I didn't think about what would happen when the content has to wrap. I'll take a look - there's probably a way to keep the flex while fixing what screenreaders do, it was just laziness from me that I thought "that flex is only there to basically right justify it, so text-align: right will work instead" without thinking of wrapping.